contracts/factories/open-edition-factory/src/contract.rs

```diff
@@ -32,6 +34,15 @@ pub fn instantiate(
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
```

Comment:
This could be written a lot cleaner. 

1. Please start using `ensure!()` for these kinds of checks.
2. Having 3+ deep calls is a code smell, so can make cleaner with `let params = msg.params`.
3. `ContractError` is imported, so no reason to duplicate it in error calls.

Reply:
fixed